index.src.html

```diff
@@ -665,15 +667,17 @@ <h3 id="parse-referrer-policy-from-header">
   <ol>
     <li>
       Let |policy-tokens| be the result of
-      parsing `<code>Referrer-Policy</code>` in |response|'s header list.
+      <a>parsing</a> `<code>Referrer-Policy</code>` in
+      |response|'s <a>header list</a>.
     </li>
     <li>
       Let |policy| be the empty string.
     </li>
     <li>
-      For each <var>token</var> in <var>policy-tokens</var>, execute
-      [[#determine-policy-for-token]] on <var>token</var> and
-      set <var>policy</var> to the result if it is not the empty string.
+      For each <var>token</var> in <var>policy-tokens</var>,
```

Comment:
Maybe while you're here you might want to consistify the algorithm between `|variable|` and `<var>variable</var>` :)